container/nginx/conf.d/shared.conf

```diff
@@ -56,14 +56,17 @@ location / {
     proxy_cache_bypass          $arg_nocache;
     proxy_no_cache              $arg_nocache;
 
-    add_header 'Saturn-Cache-Status'            $upstream_cache_status always;
+    if ($node_id) {
+        add_header 'Saturn-Node-Id'                 $node_id always;
+    }
```

Comment:
have you tested that this will not cause any issues?
https://www.nginx.com/resources/wiki/start/topics/depth/ifisevil/
http://agentzh.blogspot.com/2011/03/how-nginx-location-if-works.html

Reply:
ah, it won't work like this, thing is, it won't be available on first run 🤔

Reply:
On first run, nginx crashes with 
```
2023/01/24 15:52:01 [emerg] 34#34: unknown "node_id" variable
nginx: [emerg] unknown "node_id" variable
```

but! The container keeps running. The container should exit if nginx fails to start, since its useless if nginx is down.